paddle/fluid/operators/conv_op.cc

```diff
@@ -109,6 +109,7 @@ framework::OpKernelType ConvOp::GetExpectedKernelType(
 }
 
 void Conv2DOpMaker::Make() {
+  AddAttr<bool>("is_test", "").SetDefault(false);
```

Comment:
It might be worth to move this line to where the other attributes are defined to keep the consistency. 
Also, consider adding the docs saying `(bool, default false) Only used in mkldnn kernel`.

Reply:
This line is compatible with the content of the BatchNormOpMaker.Make() function.
Look at https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/fluid/operators/batch_norm_op.cc source code.